ex9/js/main.js

```diff
@@ -0,0 +1,38 @@
+requirejs(['ajaxGet','load','filter', 'sort'] , function(ajaxGet, load, filter, sort) {
+  $(function() {
+    var phoneBookData = {};
```

Comment:
I think your code elsewhere expects to find an array and not an object (try filtering or ordering without loading the data -- you'll probably get an Error). Initialize to `[]` instead.